applications/asset_tracker/src/main.c

```diff
@@ -608,6 +655,9 @@ static void work_init(void)
 	k_work_init(&send_flip_data_work, send_flip_data_work_fn);
 	k_delayed_work_init(&flip_poll_work, flip_send);
 	k_delayed_work_init(&long_press_button_work, accelerometer_calibrate);
+#if CONFIG_BATTERY_MONITOR
```

Comment:
Why not being consistent and use `IS_ENABLED` macro here as well? You don't need to conditionally include `battery_monitor_check` sources, the linker should handle this.